crates/cli/src/lib.rs

```diff
@@ -1306,6 +1311,7 @@ fn roc_run_executable_file_path(binary_bytes: &[u8]) -> std::io::Result<Executab
     let mut file = OpenOptions::new()
         .create(true)
         .write(true)
+        .truncate(true)
```

Comment:
was this a bug before?

Reply:
Clippy likes to see the truncate behavior explicitly defined:
```
error: file opened with `create`, but `truncate` behavior not defined
   --> crates/linker/src/lib.rs:565:10
    |
565 |         .create(true)
    |          ^^^^^^^^^^^^- help: add: `.truncate(true)`
    |
    = help: if you intend to overwrite an existing file entirely, call `.truncate(true)`
    = help: if you instead know that you may want to keep some parts of the old file, call `.truncate(false)`
    = help: alternatively, use `.append(true)` to append to the file instead of overwriting it
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#suspicious_open_options
    = note: `-D clippy::suspicious-open-options` implied by `-D warnings`
    = help: to override `-D warnings` add `#[allow(clippy::suspicious_open_options)]`

```